Scripts/Packaging/build_packages.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+# MIT License
+#
+# Copyright (c) 2022-2024 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+set -e
+
+PACKAGE_NAME="ROCm-SDK-Examples"
+PACKAGE_VERSION="6.2.0"
+PACKAGE_VENDOR="Advanced Micro Devices, Inc."
+PACKAGE_CONTACT="ROCm Developer Support <rocm-dev.support@amd.com>"
+PACKAGE_DESCRIPTION_SUMMARY="A collection of examples for the ROCm software stack"
+PACKAGE_INSTALL_PREFIX="/opt/rocm/examples"
+
+GIT_TOP_LEVEL=$(git rev-parse --show-toplevel)
+BUILD_DIR="$GIT_TOP_LEVEL/build"
+DEB_DIR="$BUILD_DIR/deb"
+RPM_DIR="$BUILD_DIR/rpm"
+RPM_BUILD_DIR="$RPM_DIR/BUILD"
+RPM_SOURCE_DIR="$RPM_DIR/SOURCES"
+RPM_SPEC_DIR="$RPM_DIR/SPECS"
+RPM_RPMS_DIR="$RPM_DIR/RPMS"
+RPM_SRPM_DIR="$RPM_DIR/SRPMS"
+
+# Directories to be included in the package
+SOURCE_DIRS=(
+    "AI"
+    "Applications"
+    "Common"
+    "Dockerfiles"
+    "External"
+    "HIP-Basic"
+    "Libraries"
+    "LLVM_ASAN"
+)
+
+copy_sources() {
+    local dest_dir=$1
+    mkdir -p $dest_dir
+
+    # Copy source files in root to package
+    cp LICENSE.md CMakeLists.txt README.md $dest_dir
+
+    # Copy source directories to package
+    for dir in "${SOURCE_DIRS[@]}"; do
+        rsync -a --exclude 'build' --exclude '.gitignore' --exclude '*.vcxproj**' --exclude '*.sln' --exclude 'bin' --exclude '*.o' --exclude '*.exe' $dir $dest_dir
+    done
+}
+
+create_deb_package() {
+    local package_dir=$1
+    local control_file="$package_dir/DEBIAN/control"
+    mkdir -p "$(dirname $control_file)"
+
+    # Create control file
+    echo "Package: $PACKAGE_NAME" > $control_file
+    echo "Version: $PACKAGE_VERSION" >> $control_file
+    echo "Architecture: amd64" >> $control_file
+    echo "Maintainer: $PACKAGE_CONTACT" >> $control_file
+    echo "Description: $PACKAGE_DESCRIPTION_SUMMARY" >> $control_file
+    echo "Homepage: https://github.com/ROCm/ROCm-examples" >> $control_file
```

Comment:
Also nit: the RPM uses this value as well, I would define a variable `PACKAGE_HOMEPAGE_URL` or something like that with this link, just to make sure that we set up always the same value for both packages.